test/functional/api/cas/cache_config.py

```diff
@@ -85,6 +85,9 @@ def from_name(cls, name):
 
         raise ValueError(f"{name} is not a valid sequential cut off name")
 
+    def __str__(self):
+        return self.value
+
```

Comment:
this is probably not needed after the refactor to string enums

Reply:
it is needed